app/static/js/Mod.js

```diff
@@ -92,4 +92,76 @@ u.addEventForChild(document, 'change', '#flair_id', function (e, qelem) {
   } else {
     document.querySelector('#assign_flair_form #text').classList.add('hide')
   }
-});
\ No newline at end of file
+});
+
+
+// Admin site configuration page
+
+u.addEventForChild(document, 'click', '.admin-config-doc-toggle', function (e, qelem) {
+    let docElem = document.getElementById(qelem.id + "-doc");
+    if (docElem.classList.contains('hide')) {
+        docElem.classList.remove('hide');
+        qelem.innerHTML = "▿";
+    } else {
+        docElem.classList.add('hide')
+        qelem.innerHTML = "▹";
+    }
+});
+
+u.addEventForChild(document, 'click', '.admin-config-edit', function (e, qelem) {
+  const errorbox = document.querySelector('.error');
+  let name = qelem.getAttribute('data-setting');
+  let type = qelem.getAttribute('data-type');
+  errorbox.classList.add('hide');
+
+  if (type == 'bool') {
+    u.post('/do/admin/config/toggle', {setting: name},
+           function (data) {
+             if (data.status != 'ok') {
+               errorbox.classList.remove('hide');
+               errorbox.innerHTML = _('Error: %1', data.error);
+               errorbox.scrollIntoView();
+             } else {
+               window.location.reload();
+             }
+           }, function () {
+             errorbox.classList.remove('hide');
+             errorbox.innerHTML = _('Could not contact the server');
+             errorbox.scrollIntoView();
+           });
```

Comment:
We should have a confirmation prompt here to prevent somebody from accidentally clicking and toggling a boolean accidentally

Reply:
Done, and in the process I realized that the True and False shown in the table were generated by Python and not translatable.  I changed them to Enabled and Disabled and now they are translatable.